Deeploy/Targets/Snitch/Platform.py

```
@@ -2,46 +2,69 @@
-SoftmaxMapper = NodeMapper(SoftmaxParser(), SnitchiSoftmaxTilingReadyBindings)
-iNoNormMapper = NodeMapper(iNoNormParser(), SnitchiNoNormTilingReadyBindings)
+# These use TiledTransformer but work in both modes (original upstream behavior)
+GemmMapper = NodeMapper(SnitchGEMMParser(), SnitchGemmBindings)
+RqGemmMapper = NodeMapper(SnitchRQGEMMParser(), SnitchRqGemmBindings)
+iSoftmaxMapper = NodeMapper(iSoftmaxParser(), SnitchiSoftmaxBindings)
+SoftmaxMapper = NodeMapper(SoftmaxParser(), SnitchiSoftmaxBindings)
+iNoNormMapper = NodeMapper(iNoNormParser(), SnitchiNoNormBindings)
 iLayerNormMapper = NodeMapper(iLayerNormParser(), BasicLayerNormBindings)
-RQAddMapper = NodeMapper(RQAddParser(), SnitchRQAddTilingReadyBindings)
-AddMapper = NodeMapper(AddParser(), SnitchAddTileReadyBindings)
+RQAddMapper = NodeMapper(RQAddParser(), SnitchRQAddBindings)
```

Ruff (0.14.14)

[error] Redefinition of unused `RQAddMapper` from line 25: `RQAddMapper` redefined here

(F811)